src/tests/Common/Directory.Build.targets

```diff
@@ -129,6 +129,32 @@
           TargetDir="runtime-test/"/>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetOS)' == 'Android'">
+        <RunTimeDependencyCopyLocal
+          Include="@(RuntimeFiles)"
+          TargetDir="runtimepack/native/"/>
+
+        <RunTimeDependencyCopyLocal
```

Comment:
These are also needed for Android? The comment for wasm made it seem like a special case.

Reply:
Android and wasm fall into the same special case category. So yes, they are needed.